### PR TITLE
mon: fix two stray legacy get_health() callers

### PR DESCRIPTION
**Pull request by liewegas**
These need to call the new method when appropriate.

Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

File: src/mon/Monitor.cc [hunk 1]
```diff
@@ -4400,8 +4404,13 @@ void Monitor::handle_ping(MonOpRequestRef op)
   boost::scoped_ptr<Formatter> f(new JSONFormatter(true));
   f->open_object_section("pong");
 
-  list<string> health_str;
-  get_health(health_str, NULL, f.get());
+  if (osdmon()->osdmap.require_osd_release >= CEPH_RELEASE_LUMINOUS) {
```
Comment (tchaikov) [inline src/mon/Monitor.cc]: i wonder if there is any daemon read the payload of `MPing`? can we just drop the code to stuff the reply?

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2017-08-27_16:28:46-rados-wip-kefu-testing-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2017-08-28_15:54:18-rados-wip-kefu-testing-distro-basic-smithi/

File: src/mon/Monitor.cc (hunk 1 above)
Comment (liewegas) [reply src/mon/Monitor.cc]: Yeah, no users.  There was clearly some intent but it isn't implemented.  @jecluis, any objection to just removing this?  @tchaikov he may still be on vacation, in which case let's leave this for now and follow up when he gets back?

Comment (tchaikov) [reply src/mon/Monitor.cc]: sure. then this change lgtm.

**Pull request merged**